package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lumen5/framefusion",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
I don't see a tag for 1.0.8 in framefusion?

Reply:
oh wait this is the framefusion repo, I get it... 😅

Reply:
it's here: https://www.npmjs.com/package/@lumen5/framefusion

Reply:
we need to run `npm publish`

Reply:
Did the ci job fail last time then?